<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="surapaneni" ignored="0" name="Test Test2" tests="4" failures="1" timestamp="2020-12-11T17:43:49 IST" time="0.094" errors="0">
  <testcase name="doLogin" time="0.012" classname="udemyTestNG.Testcase2">
    <failure type="java.lang.AssertionError" message="login failed">
      <![CDATA[java.lang.AssertionError: login failed
at udemyTestNG.Testcase2.doLogin(Testcase2.java:12)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase name="goToHomePage" time="0.0" classname="udemyTestNG.Testcase2">
    <skipped/>
  </testcase> <!-- goToHomePage -->
  <testcase name="thirdTest" time="0.003" classname="udemyTestNG.Testcase2"/>
  <testcase name="fourthTest" time="0.009" classname="udemyTestNG.Testcase2"/>
</testsuite> <!-- Test Test2 -->
